build.gradle

```diff
@@ -64,8 +64,8 @@ dependencies {
 
 	testCompile 'junit:junit:4.12'
 	testCompile 'org.apache.derby:derby:10.12.1.1'
-	testCompile 'org.hamcrest:hamcrest-all:1.3'
-	testCompile 'org.mockito:mockito-all:2.0.2-beta'
+	testCompile 'org.hamcrest:java-hamcrest:2.0.0.0'
+	testCompile 'org.mockito:mockito-core:2.0.82-beta'
```

Comment:
Updated mockito in Order to fix something which didn't work... Kept for the any() thing...
Hamcrest update is just btw